libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/DataCharacteristics.tsx

```diff
@@ -25,13 +26,26 @@ export interface IDataCharacteristicsProps {
 }
 export interface IDataCharacteristicsState {
   columnCount: number[];
-  items: Map<string, IVisionListItem[]>;
-  labels: string[];
-  labelVisibilities: Map<string, boolean>;
+  dropdownOptionsP: IDropdownOption[];
```

Comment:
nit: maybe use full names instead of P/T to be clearer (so it's easier to read).  My guess is P = Predicted, T = True Y?